localisation/polish/effects_l_polish.yml

```diff
@@ -438,14 +438,14 @@
  EFFECT_UPGRADE_INTELLIGENCE_AGENCY:0 "Použijte vylepšení $UPGRADE|H$ na svou zpravodajskou agenturu."
  EFFECT_WHITE_PEACE_WITH:1 "Bílý mír s $COUNTRY|Y$."
  IDEA_EXPIRED_DESC:0 "Nedostáváme již efekty z $IDEA|H$."
- IDEA_EXPIRED_HEADER:0 "Myšlenka vypršela"
+ IDEA_EXPIRED_HEADER:0 "Idea vypršela"
  IDEA_EXPIRED_TT: "$IDEA|H$ vypršela. Poskytovala následující účinky:\n$EFFECTS$"
- NO_EFFECT:0 "Neděje se nic"
+ NO_EFFECT:0 "Žádný efekt"
  POLITICS_ADD_FUEL:0 "Palivo: $VAL|=+0$."
  POLITICS_ADD_POLITICAL_POWER:0 "Politická síla: $VAL|=+0$."
- POLITICS_PARTY_NAME_CHANGE:0 "Strana $IDEOLOGY|Y$ se nyní bude nazývat '$NAME|Y$'."
- POLITICS_SET_POLITICAL_POWER:0 "Nastavte politickou moc na $VAL|0Y$."
- RESEED_DIV_COMMANDER_EFFECT:0 "Nastavení semene velitele na $SEED$ (Debug)"
+ POLITICS_PARTY_NAME_CHANGE:0 "Nyní se bude $IDEOLOGY|Y$ strana nazývat '$NAME|Y$'."
+ POLITICS_SET_POLITICAL_POWER:0 "Nastavte politickou sílu na $VAL|0Y$."
+ RESEED_DIV_COMMANDER_EFFECT:0 "Nastavení seedu velitele na $SEED$ (Debug)"
```

Comment:
toto sa niekedy prekladá ako semínko, v kľude môže byť seed, to je taký herný slang